NAMESPACE

```diff
@@ -1,13 +1,16 @@
 # Generated by roxygen2: do not edit by hand
 
 S3method(augment,multinom)
+S3method(augment,xgb.Booster)
+S3method(glance,xgb.Booster)
+S3method(tidy,xgb.Booster)
 export("%>%")
 export("%nin%")
+export()
```

Comment:
Somehow, it's generated by devtools::check() and this file says "# Generated by roxygen2: do not edit by hand" so probably we shouldn't remove it